client/src/modules/events/pages/eventPage.tsx

```diff
@@ -100,6 +101,14 @@ export const EventPage: NextPage = () => {
         userIds={data?.event?.rsvps.map((r) => r.user.id) || []}
         modalProps={modalProps}
       />
+      <Image
+        boxSize="100%"
+        maxH="300px"
+        src={data.event.image}
+        alt="The freeCodeCamp logo"
```

Comment:
Can we add the alt text when adding the image and then use it here?

Reply:
Sure a field called description should do it.

Reply:
Thinking about it, this is purely decorative. We can just require `alt=""` since screenreaders can safely ignore it.